components/Card/index.js

```diff
@@ -0,0 +1,75 @@
+import { useAtom } from "jotai";
+import { useState } from "react";
+
+export default function Card({ initialItems, items }) {
```

Comment:
The props `initialItems` and `item` have slightly confusing names.

`initialItems` describes the list of items that are currently selected.
`items` describes the list of the items that are selectable.

is this correct?